File: 231-power-of-two/231-power-of-two.js
/**
 * @param {number} n
 * @return {boolean}
 */
var isPowerOfTwo = function(n) {
    if (n === 0) return false;
    if (n === 1) return true; 
    let i = 0;
    let power = 1
    while (power < n){
        power = power * 2;
    }
    if(power === n)return true;
    else return false;
};

File: 231-power-of-two/231-power-of-two.rb
# @param {Integer} n
# @return {Boolean}
def is_power_of_two(n)
    if n === 0 
        false 
    end 
    
    if n === 1 
        true 
    end 
    
    power = 1
    while power < n
        power = power * 2
    end 
    
    if power === n 
        true 
    else 
        false 
    end 
end
